rel/apps/couch_epi.config

```diff
@@ -0,0 +1,9 @@
+{plugins, [
+    couch_db_epi,
+    chttpd_epi,
+    couch_index_epi,
+    global_changes_epi,
+    mango_epi,
+    mem3_epi,
+    setup_epi
```

Comment:
Hm...I cannot apply patch on current master as well. But if I checkout you branch, sys.config is there. May squash all the reverts?